syntax = "proto3";

service MasterNode {
  rpc HealthCheck(EmptyRequest) returns (HealthCheckResponse) {}
}

message EmptyRequest {}

message HealthCheckResponse {
  string message = 1;
}
